src/language-js/needs-parens.js

```diff
@@ -801,6 +807,63 @@ function isStatement(node) {
   );
 }
 
+function includesFunctionTypeInObjectType(path) {
+  const node = path.getValue();
+  if (!node) {
+    return false;
+  }
+
+  switch (node.type) {
+    case "ObjectTypeAnnotation": {
+      let isIncludeFunctionType = false;
```

Comment:
"is include" is ungrammatical. Please change it to "includes" everywhere.

Reply:
Thanks